src/OrchardCore/OrchardCore.ContentManagement.Display/ContentDisplay/ContentItemDisplayCoordinator.cs

```diff
@@ -85,6 +85,14 @@ public async Task BuildDisplayAsync(ContentItem contentItem, BuildDisplayContext
                 }
             }
 
+            var settings = contentTypeDefinition?.GetSettings<ContentTypeSettings>();
+            var stereotype = "";
```

Comment:
If this is only used for the dynamic part shapes, it should probably be moved to inside the if block. i.e. L149

No need to resolve it any earlier, if it doesn't need resolving. 

I can't see anywhere else it is being used

Reply:
It gets `stereotype` of `ContentType`  - stays the same for every part, no need to call `GetSettings` inside foreach in loop